README.md

```diff
@@ -0,0 +1,72 @@
+# Installation
+
+You need to install the following packages in order to use ebay skin:
+
+```
+yarn add @ebay/skin @ebay/browserslist-config
```

Comment:
But that would install these packages globally, right? We only need to install them locally to the project, so having them listed in dependencies and running `yarn` should suffice. So the only install step is to run yarn.

Reply:
Oh, I see that you are documenting this from the perspective of adding to the react starter app. In that case just adding the `-D` should be good.